api/src/main/java/org/pmiops/workbench/captcha/CaptchaServerVerificationResponse.java

```diff
@@ -0,0 +1,93 @@
+package org.pmiops.workbench.captcha;
```

Comment:
Opt: Could either of these files be generated with Swagger? Assuming I understand and this is an api response object and a caller to the captcha API?

This is opt because this is a small number of methods and is unlikely to change, but may be worth considering.

Reply:
I started with the swagger approach that did not work for me hence i used this.

Reply:
Apparently i need to borrow Greg's shame cube, the reason why this was not working in swagger was spelling mistake and security Definitions (we do not need it here)